src/vg_active_segment.erl

```diff
@@ -78,13 +120,52 @@ init([Topic, Partition, NextBrick, Tab]) ->
 
 handle_call({write, MessageSet}, _From, State=#state{topic=_Topic,
                                                      partition=_Partition,
-                                                     next_brick=_NextBrick}) ->
+                                                     next_brick=NextBrick}) ->
     {FirstID, State1} = write_message_set(MessageSet, State),
-    {reply, {ok, FirstID}, State1}.
+    lager:info("~p XXNEXT ~p", [node(), NextBrick]),
+    case NextBrick of
+        solo -> ok;
+        tail -> ok;
+        last -> ok;
+        _ ->
+            %% this will be a bottleneck, but OK for now
+            %% does gproc's via work on remote nodes?
+            R = gen_server:cast(NextBrick,
```

Comment:
did a cast here for historical reasons (originally I did not have the pid).  could change it to a straight message send if we wanted.